modules/basics/src/main/java/com/opengamma/strata/basics/index/IborIndex.java

```diff
@@ -81,6 +83,24 @@ public static IborIndex of(String uniqueName) {
   public abstract HolidayCalendar getFixingCalendar();
 
   /**
+   * Gets the fixing time of the index.
```

Comment:
Remove these two methods from `IborIndex` and add `calculateFixingDateTime(LocalDate)` instead (returning `ZonedDateTime`). This avoids exposing the data over the `IborIndex` interface.